### ISSUE TITLE
FR: redirect traling slashes

### ISSUE THREAD

**Issue by olivroy** [etiennebacher/altdoc#281]
For example, make this https://vincentarelbundock.github.io/tinytable/news/ a valid url.

https://vincentarelbundock.github.io/tinytable/NEWS works as expected.

I have a routine to check news on online doc. the first link works for pkgdown sites. I wondered if it were possible to make it work for altdoc sites.

**Comment by vincentarelbundock** [comment]
maybe the tinytable website has changed, but it looks like the address should be NEWS.html

Do you have another example?

Also, that might be a Quarto issue rather than altdoc...

**Comment by olivroy** [comment]
Possibly, but also the fact that https://vincentarelbundock.github.io/tinytable/NEWS works but not https://vincentarelbundock.github.io/tinytable/news is a bit strange?

**Comment by vincentarelbundock** [comment]
yep, good point. But honestly, I'm not sure what `altdoc` can do about this. Case sensitivity of links seems like it would be in scope for Quarto itself. Can I change anything in the `_quarto.yml` to force Quarto to disregard upper cases? I have no idea...

**Comment by olivroy** [comment]
Not sure. https://github.com/quarto-dev/quarto-cli/issues/6848 is opened about case sensitiviry

**Comment by vincentarelbundock** [comment]
Altdoc allows redirects because we can specify them manually in the `.yml` file. 

We are only prepping files for Quarto, not writing the HTML, so this feels like it is in Quarto's scope.

Closing now, but Etienne should feel free to re-open if he wants to do something about this.
